lang/gcc10-bootstrap/Portfile

```diff
@@ -10,7 +10,7 @@ name                gcc10-bootstrap
 
 # Note, this is the last version of gcc which haven't required c++11 for bootstrap
 version             10.3.0
-revision            1
+revision            2
```

Comment:
I think we don't need another revbump yet. This should just fix all the systems that didn't build before.

We don't want to make the buildbots stroke out here :>

Reply:
@kencu very good point!